packages/terra-hookshot/examples/index-examples/BoundedHookshotExample.jsx

```diff
@@ -0,0 +1,162 @@
+import React from 'react';
```

Comment:
should this file include the `.example` in its file name @emilyrohrbough ? e.g. `BoundedHookshot.example.jsx`

Reply:
This look like they are index-examples for the site page pulled in by Index.jsx, not text example so they should be ok.

Reply:
Ok :+1: